### ISSUE TITLE
Change Visual maping

### ISSUE THREAD

**Issue by vhoyer** [philc/vimium#2305]
Its an issue because, Brazilian keyboards have the __ç__ and this single key, changes the way your brain works lol, making using the movement from __hjkl__ keys really annoying to use, I have faith i'm not the only Brazilian who changes the movement biding to __jklç__, so I'd really love to be able to change this :D

**Comment by smblott-github** [comment]
Thanks, @vhoyer.  This has been suggested and requested before, but remarkably few times.  I'm not too keen on adding:

    vmap ç scrollRight

and the like.  Everything in visual mode is pretty hard-wired at the moment, and it would be a big change for a fairly small benefit.  We'd have to add command names, descriptions and a whole new suite of help-page infrastructure.

What about something like this?

    translate ç l

Perhaps in effect for normal mode and visual mode only?  Or universally?

The meaning would be: *whenever you see "ç", replace it with "l"*.  No smarts.  Documented on the wiki only.

**Comment by vhoyer** [comment]
I didn't quite understand, the `translate` option isn't available right now, right? That's the interpretation I've got. But, if this is not the case, I've just tried this command and it didn't work, If you are planning on adding it, this would be great, other than that, I'm pretty happy with this extension, It was quite annoying switching from vim to chrome and having my mind broken for a sec, kk, thank you vary much.
